### PR TITLE
[RFR] Add sub menu and first comming

### PR DESCRIPTION
**Pull request by BaptistePDJ**

### CODE REVIEW

File: project/templates/client/index.html.twig [hunk 1]
```diff
@@ -34,51 +34,56 @@
...
+                        <th><a class="a-ordered" href="{{ path('client_ordered', {'orderedType' : 'zipCode', 'orderBy' : app.request.get('orderedType') is same as ('zipCode') ? sort, 'id' : currentClient.id }) }}">Code postal <i class="fa fa-arrow-{{ (sort is same as ('ASC') and app.request.get('orderedType') is same as ('zipCode')) ? 'down' : 'up' }}"></i></a></th>
+                        <th><a class="a-ordered" href="{{ path('client_ordered', {'orderedType' : 'phone', 'orderBy' : app.request.get('orderedType') is same as ('phone') ? sort, 'id' : currentClient.id }) }}">Téléphone <i class="fa fa-arrow-{{ (sort is same as ('ASC') and app.request.get('orderedType') is same as ('phone')) ? 'down' : 'up' }}"></i></a></th>
+                        <th><a class="a-ordered" href="{{ path('client_ordered', {'orderedType' : 'enable', 'orderBy' : app.request.get('orderedType') is same as ('enable') ? sort, 'id' : currentClient.id }) }}">Actif <i class="fa fa-arrow-{{ (sort is same as ('ASC') and app.request.get('orderedType') is same as ('enable')) ? 'down' : 'up' }}"></i></a></th>
+                        <th>Voir</th>
                     </tr>
```
Comment (BaptistePDJ) [inline project/templates/client/index.html.twig]: indentation

File: project/templates/client/index.html.twig [hunk 2]
```diff
@@ -34,51 +34,56 @@
...
+                            <td>{{ client.address }}</td>
+                            <td>{{ client.city }}</td>
+                            <td>{{ client.zipCode }}</td>
+                            <td>{{ client.phone }}</td>
+                            <td>{{ client.enable ? 'Oui' : 'Non' }}</td>
+                            <td style="width: 17%">
+                                <a href="{{ path('client_show', {'id': client.id}) }}" class="btn btn-secondary">Détails</a>
+                            </td>
+                        </tr>
+                    {% endfor %}
```
Comment (BaptistePDJ) [inline project/templates/client/index.html.twig]: indentation

File: project/templates/domain_name/index.html.twig [hunk 3]
```diff
@@ -33,56 +33,61 @@
...
+                            <td>{{ domain_name.renewalType }}</td>
+                            <td>{{ domain_name.startDate ? domain_name.startDate|date('Y-m-d') : '' }}</td>
+                            <td>{{ domain_name.renewalDate ? domain_name.renewalDate|date('Y-m-d') : '' }}</td>
+                            <td>{{ domain_name.site is defined and domain_name.site is not null ? domain_name.site.name : '-' }}</td>
+                            <td>{{ domain_name.enable ? 'Oui' : 'Non' }}</td>
+                            <td>
+                                <a href="{{ path('domain_name_show', {'id': domain_name.id}) }}" class="btn btn-secondary">Détails</a>
+                            </td>
+                        </tr>
+                    {% endfor %}
```
Comment (BaptistePDJ) [inline project/templates/domain_name/index.html.twig]: indentation

File: project/templates/server/index.html.twig [hunk 4]
```diff
@@ -34,50 +34,56 @@
...
+                            <td>{{ server.cost }}</td>
+                            <td>{{ server.invoicedPrice }}</td>
+                            <td>{{ server.renewalType }}</td>
+                            <td>{{ server.renewalDate ? server.renewalDate|date('d-m-Y') : '' }}</td>
+                            <td>{{ server.enable ? 'Oui' : 'Non' }}</td>
+                            <td>
+                                <a href="{{ path('server_show', {'id': server.id}) }}" class="btn btn-secondary" >Détails</a>
+                            </td>
+                        </tr>
+                    {% endfor %}
```
Comment (BaptistePDJ) [inline project/templates/server/index.html.twig]: indentation

File: project/templates/site/index.html.twig [hunk 5]
```diff
@@ -47,43 +47,49 @@
...
+                        <tr style="vertical-align: middle;">
+                            <td>{{ site.name }}</td>
+                            <td>{{ site.onlineDate ? site.onlineDate|date('d-m-Y') : '' }}</td>
+                            <td>{{ site.online ? 'Oui' : 'Non' }}</td>
+                            <td>{{ site.enable ? 'Oui' : 'Non' }}</td>
+                            <td>
+                                <a href="{{ path('site_show', {'id': site.id}) }}" class="btn btn-secondary">Détails</a>
+                            </td>
+                        </tr>
+                    {% endfor %}
```
Comment (BaptistePDJ) [inline project/templates/site/index.html.twig]: indentation

**Review by julienmilazzo — APPROVED** [review]

**Pull request merged**